eslintrc.json

```diff
@@ -43,6 +44,8 @@
     "react/no-danger": [0],
     "react/no-string-refs": [0],
     "react/no-unescaped-entities": [0],
+    "react/no-unused-prop-types": [2, {"skipShapeProps": true}],
+    "react/prop-types": [2, { "ignore": ["className"] }],
```

Comment:
I think className should be included here... it was excluded in Axiom because of the removed HOC approach. When the Base component was added and the HOC pattern was removed, this should have been removed too.

Reply:
will remove 👍